docs/Xamarin.Forms.Core/Xamarin.Forms/Button.xml

```diff
@@ -21,6 +21,9 @@
     <Interface>
       <InterfaceName>Xamarin.Forms.IElementConfiguration&lt;Xamarin.Forms.Button&gt;</InterfaceName>
     </Interface>
+    <Interface>
+      <InterfaceName>Xamarin.Forms.IFontElement</InterfaceName>
+    </Interface>
```

Comment:
This changes are automatically generated by  'monodoc' with latest code. I'll update if this change is not neccessray.